asset/css/markbind.css

```diff
@@ -23,6 +23,12 @@ pre > code.hljs {
     outline: none !important;
 }
 
+/* Bootstrap responsive container class */
+.container-fluid {
+    max-width: 1200px;
```

Comment:
Why are we restricting the maximum width to 1200px?

Reply:
Prof mentioned the content is difficult to read when it is too stretched out